sogs/routes/auth.py

```diff
@@ -123,6 +123,41 @@ def required_user_wrapper(*args, **kwargs):
     return required_user_wrapper
 
 
+def require_mod(room, *, admin=False):
+    """Checks a room for moderator or admin permission; aborts with 401 Unauthorized if there is no
+    user in the request, and 403 Forbidden if g.user does not have moderator (or admin, if
+    specified) permission."""
+    require_user()
+    if not (room.check_admin(g.user) if admin else room.check_moderator(g.user)):
+        abort_with_reason(http.FORBIDDEN, 'This endpoint requires moderator permissions')
```

Comment:
error string should contain 'admin' not 'moderator' when it's looking for admin not moderator

Reply:
Fixed.